Automattic/sensei-wc-paid-courses-language-packs
# Include JSON language files in built packages

### Changes Proposed

- Uses `wp-cli` to include the JSON files in the built packages.
- Adds composer dependency for wp-cli (need to run `composer install` when you switch to PR).
- Updates 2.3.0 packages with JSON files.

### Testing Instructions
- On 586-gh-Automattic/sensei-wc-paid-courses, use one of the generated packages in 2.3.0 and make sure Sensei sidebar is loaded when editing a course.
